### DISCUSSION TITLE
Bull MQ vs Trigger Dev?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [taskforcesh/bullmq#2458] [Q&A]
My simple application right now only needs 1 function (to send email after payment is done) & I think Bull MQ would be fine. What do you think? I have never used Queues before so don't know much about it.

Also, would love to know what are the benefits of using Bull MQ over Trigger Dev?

In what use-cases would Bull MQ be preferred over Trigger Dev?

Is Bull MQ Pro similar to Trigger Dev?

**Comment by manast** [comment]
For you simple use case BullMQ would be more than enough. Regarding Trigger Dev I have not done any research so I do not know what they offer... but when it is about hosted databases, I think it is safer to rely on battle tested technologies such as Redis vs the new shinny guy in the block.
